client/src/App.scss

```diff
@@ -4,7 +4,7 @@
 main > article {
 	padding: calc(#{layout.$baseline} * 4);
 
-	> h2 {
```

Comment:
It seems little dangerous to update a heading structure in `App.scss`. 
It can lead to potential styling issues and I noticed that the Drafts page was not updated to match this new structure.

How about 
- update the Drafts page to use the new heading structure for consistency?
- create a reusable `PageTitle` component to ensure consistent heading structure and styling across all pages?